### ISSUE TITLE
TypeLoadException when trying to load QueryCollection in .NET 6 isolated function

### ISSUE THREAD

**Issue by RyanMarcotte** [Azure/azure-functions-openapi-extension#534]
Similar to #296 , this error is thrown when using version 1.5.0 of `Microsoft.Azure.Functions.Worker.Extensions.OpenApi`.  Reverting to 1.4.0 fixed the problem for me.

```
[2023-01-09T03:00:55.900Z] An error occurred while processing the request (`RenderSwaggerUI`).
[2023-01-09T03:00:55.903Z] Result: An error occurred while processing the request (`RenderSwaggerUI`).
Exception: System.TypeLoadException: Could not load type 'Microsoft.AspNetCore.Http.Internal.QueryCollection' from assembly 'Microsoft.AspNetCore.Http, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'.
[2023-01-09T03:00:55.917Z]    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Extensions.OpenApiHttpRequestDataExtensions.Queries(HttpRequestData req)
[2023-01-09T03:00:55.937Z]    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.HttpRequestObject..ctor(HttpRequestData req)
[2023-01-09T03:00:55.940Z]    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Functions.OpenApiTriggerFunction.RenderSwaggerUI(HttpRequestData req, FunctionContext ctx)
[2023-01-09T03:00:55.954Z]    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.DefaultOpenApiHttpTrigger.RenderSwaggerUI(HttpRequestData req, FunctionContext ctx)
Stack:    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Extensions.OpenApiHttpRequestDataExtensions.Queries(HttpRequestData req)
[2023-01-09T03:00:55.964Z]    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.HttpRequestObject..ctor(HttpRequestData req)
[2023-01-09T03:00:55.973Z]    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Functions.OpenApiTriggerFunction.RenderSwaggerUI(HttpRequestData req, FunctionContext ctx)
[2023-01-09T03:00:55.977Z]    at Microsoft.Azure.Functions.Worker.Extensions.OpenApi.DefaultOpenApiHttpTrigger.RenderSwaggerUI(HttpRequestData req, FunctionContext ctx).
```

Guessing that this issue is due to changes made to [OpenApiHttpRequestDataExtensions.cs](https://github.com/Azure/azure-functions-openapi-extension/pull/522/files#diff-5ad3cebc7d7f9186992b8ca38958975fec4d8196cc1d93ee305bd7cd0654138c) recently.

**Comment by holdenmai** [comment]
@RyanMarcotte I have some of my findings around how I fixed this without downgrading on #536

**Comment by justinyoo** [comment]
@RyanMarcotte Thanks for the issue! Let's discuss on #536
